h2o-genmodel/src/main/java/hex/genmodel/algos/targetencoder/EncodingMaps.java

```diff
@@ -0,0 +1,51 @@
+package hex.genmodel.algos.targetencoder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class EncodingMaps {
```

Comment:
Does this no need to be serializable? In SW MOJOs are serialized and send over the wire.

Please add tests for Serializing/Deserializing the instance of a loaded mojo.

Reply:
@michalkurka I don't see any methods for serialisation of a loaded instance of mojo. Could you please elaborate on this?  If you meant integration test which tests `write then load` path - I have it. `TEMojoIntegrationTest.withoutBlending()` for example. Anyway this class is not being used for direct serialisation.  This is just a human readable wrapper.

Reply:
Needs to be Serializable because it is used in MojoModel which implements Serializable - please add tests

you can check EasyPredictModelWrapperTest - there are some serialization tests

Reply:
@michalkurka Did not see use case in h2o3 and also was confused that we need to use Serializable when we have our custom Freezable approach. It probably means that we do not want to force production users to rely on our custom serialisation. Got it , fixed and added tests cc @tomkraljevic